web_src/js/markup/tasklist.js

```diff
@@ -55,12 +55,11 @@ export function initMarkupTasklist() {
           const updateUrl = editContentZone.getAttribute('data-update-url');
           const context = editContentZone.getAttribute('data-context');
 
-          await $.post(updateUrl, {
-            ignore_attachments: true,
-            _csrf: window.config.csrfToken,
-            content: newContent,
-            context
-          });
+          const requestBody = new FormData();
+          requestBody.append('ignore_attachments', 'true');
+          requestBody.append('content', newContent);
+          requestBody.append('context', context);
+          await POST(updateUrl, {data: requestBody});
```

Comment:
```suggestion
          await POST(updateUrl, {body: requestBody});
```

Although "data" also works, literally, it should use "body" here (the "fetch.js" using data as body as a fallback seems a little strange).

Just a nit.

Reply:
I tend to disagree, `data` is the primary property meant for external consumption and it's documented in the comment:

https://github.com/go-gitea/gitea/blob/1c14cd0c43d670fef984068e2666641ea5a062db/web_src/js/modules/fetch.js#L8C60-L8C64